testrail/client.py

```diff
@@ -172,13 +183,23 @@ def active_plans(self):
     # Run Methods
     @methdispatch
     def runs(self):
-        return map(Run, self.api.runs(self._project_id))
+        return RunContainer(list(map(Run, self.api.runs(self._project_id))))
 
     @runs.register(Milestone)
     def _runs_for_milestone(self, obj):
         return RunContainer(filter(
             lambda r: r.milestone.id == obj.id, self.runs()))
 
+    @runs.register(Active)
+    def _runs_for_active(self, obj):
```

Comment:
This will however also mess with the cache.  It was easy to say give me all runs and cache them.  Now you are saying give me only runs with start date x and end date y.  If there is a way to get all results I still prefer that and then filter later on the cache.